src/Tempest/View/View.php

```diff
@@ -4,18 +4,33 @@
 
 namespace Tempest\View;
 
+/**
+ * @method include(string $path, ...$params): string
```

Comment:
This sucks, will need to refactor so that the view can include another view, and the renderer knows how to render it.